### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by razacodespython**
Added:
1. readme
2. deploy.yaml file

### CODE REVIEW

File: thirdweb/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+version: "2.0"
+services:
+  web:
+    image: "thirdweb/starter-kits:latest"
```
Comment (andy108369) [inline thirdweb/deploy.yaml:4]: Please change this tag to a specific version. Akash aggressively caches them so users will get a different ":latest" tag.

File: thirdweb/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Getting Started
```
Comment (andy108369) [inline thirdweb/README.md:1]: Please also update the top level README pointing to this directory.

**Review by andy108369 — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please address couple of minor issues before we merge it :)

File: thirdweb/README.md (hunk 2 above)
Comment (kquirapas) [reply thirdweb/README.md:1]: Hey brother, I'm Kristian, Developer Advocate from thirdweb I'll be taking over this PR. May I know what you mean by this revision?

**Comment by kquirapas** [comment]
Hey @andy108369 , I'm Kristian, Developer Advocate from thirdweb. I will be taking over this PR and complete this collaboration with Akash. Nice to meet you brother!

File: thirdweb/deploy.yaml (hunk 1 above)
Comment (kquirapas) [reply thirdweb/deploy.yaml:4]: Hey Andy, checked our docker image over here: https://hub.docker.com/r/thirdweb/starter-kits/tags

We realized we only have one tag "latest" do we still have to specify a version if we only have one tag available?

image

**Review by troian — CHANGES_REQUESTED** [review]
.DS_Store files should be git ignored

File: thirdweb/README.md (hunk 2 above)
Comment (andy108369) [reply thirdweb/README.md:1]: Hi, you need to update the main README as well.
This one https://github.com/ovrclk/awesome-akash/blob/master/README.md

**Comment by kquirapas** [comment]
Decided to create a new PR at #350 so I don't have to disturb @razacodespython 

Suggesting we close this

**Comment by kquirapas** [comment]
Hey @andy108369 thirdweb PR merged. Kindly close this PR

**Pull request closed without merging**